x-pack/legacy/plugins/ml/public/components/navigation_menu/tabs.tsx

```diff
@@ -17,12 +17,8 @@ interface Props {
   tabId: TabId;
 }
 
-function getTabs(tabId: TabId, disableLinks: boolean): Tab[] {
-  const TAB_MAP: Record<TabId, Tab[]> = {
-    // overview: [],
-    datavisualizer: [],
-    data_frames: [],
-    data_frame_analytics: [],
+export function getTabs(tabId: TabId, disableLinks: boolean): Tab[] {
```

Comment:
maybe this whole  function should be a `switch`?

Reply:
I'll leave that to a possible follow up. Since `getTabs()` itself returns `[]` in cases where `tabId` is something that's not part of `TAB_MAP` there's no work-around needed when `.map()` is used later on.